root/user/profile.tt

```diff
@@ -91,11 +91,15 @@
             [% END %]
         [% END %]
 
-        [% IF c.user_exists AND (viewing_own_profile OR c.user.is_account_admin);
-             WRAPPER property name=l('Last login:');
-               UserDate.format(user.last_login_date);
-             END;
-           END %]
+        [% IF c.user_exists AND (viewing_own_profile OR c.user.is_account_admin) %]
+           [% WRAPPER property name=l('Last login:') %]
```

Comment:
Why did you change this whole section to go constantly back and forth between TT instructions and pass-through text, cluttering both the source file (with `[% %] [% %]`) and the output (with lots of spaces)?